spec/unit/support/matchers/be_a_response_like_spec.rb

```diff
@@ -17,90 +17,98 @@
     it 'fails when comparing an integer' do
       expect {
         expect({ 'a' => 1 }).to be_a_response_like({ 'a' => /\d+/ })
-      }.to raise_expectation_not_met_with_keys(bad_keys: ['a'])
+      }.to raise_expectation_not_met_with_summary_parts('! a:', '- /\d+/', '+ 1')
```

Comment:
@ericpromislow is there a slightly more understandable name for this method (`raise_expectation_not_met_with_summary_parts`)? As someone looking at these assertions for the first time, it's hard to figure out what's actually being tested without looking closely at the method implementation itself.